features/_version.feature

```diff
@@ -0,0 +1,18 @@
+Feature: UA is expected version
+
+    @series.all
+    @check.version
```

Comment:
I think we can instead use the @uses.config.check_version decorator which will automatically handle the skip logic if that attribute is not provided via `UACLIENT_BEHAVE_CHECK_VERSION=blah`